cmake/Options.cmake

```diff
@@ -57,6 +57,7 @@ function(set_options)
     option(OPTION_USE_ASSERTS "Use assertions; leave enabled for nightlies, betas, and RCs" ON)
     option(OPTION_USE_THREADS "Use threads" ON)
     option(OPTION_USE_NSIS "Use NSIS to create windows installer; enable only for stable releases" OFF)
+    option(OPTION_TOOLS_DOCS "Build only tools and docs targets" OFF)
```

Comment:
weird name for the option honestly. I wouldn't guess what it does based on that. Can't we consider "docs" to be a tool too? `OPTION_TOOLS_ONLY` sounded so much better :) Maybe others have good ideas :)